metricbeat/module/system/raid/_meta/fields.yml

```diff
@@ -8,23 +8,44 @@
       type: keyword
       description: >
         Name of the device.
-    - name: activity_state
+    - name: status
       type: keyword
       description: >
         activity-state of the device.
+    - name: sync_action
+      type: keyword
+      description: >
+        Current sync action, if the RAID array is redundant 
     - name: disks.active
       type: long
       description: >
          Number of active disks.
+    - name: disks.Unknown
+      type: long
+      description: >
+         Number of disks in a state that does not belong to any other category
```

Comment:
Not sure if I would store a count of this in the event. I'd say this is unexpected and would be enough with logging it at the debug level to investigate the value and add it to one of the existing groups.

Reply:
Ah, good point.